designs/nodepool-termination-grace-period.md

```diff
@@ -0,0 +1,127 @@
+# NodePool Termination Grace Period
```

Comment:
Thanks for this RFC! We've gone back and forth on a bunch of options (in comments) for the spec.template, spec.disruption, or just spec. Can you lay out those options, their pros and cons, and then make a recommendation?

Reply:
Updated with a preferred and alternative spec for the location of the `terminationGracePeriod` field.